src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -36,74 +38,105 @@ const propTypes = {
 class ACHContractStep extends React.Component {
     constructor(props) {
         super(props);
+        this.validate = this.validate.bind(this);
 
         this.addBeneficialOwner = this.addBeneficialOwner.bind(this);
         this.submit = this.submit.bind(this);
 
         this.state = {
             ownsMoreThan25Percent: ReimbursementAccountUtils.getDefaultStateForField(props, 'ownsMoreThan25Percent', false),
             hasOtherBeneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'hasOtherBeneficialOwners', false),
-            acceptTermsAndConditions: ReimbursementAccountUtils.getDefaultStateForField(props, 'acceptTermsAndConditions', false),
-            certifyTrueInformation: ReimbursementAccountUtils.getDefaultStateForField(props, 'certifyTrueInformation', false),
             beneficialOwners: ReimbursementAccountUtils.getDefaultStateForField(props, 'beneficialOwners', []),
         };
-
-        // These fields need to be filled out in order to submit the form (doesn't include IdentityForm fields)
-        this.requiredFields = [
-            'acceptTermsAndConditions',
-            'certifyTrueInformation',
-        ];
-
-        // Map a field to the key of the error's translation
-        this.errorTranslationKeys = {
-            acceptTermsAndConditions: 'common.error.acceptedTerms',
-            certifyTrueInformation: 'beneficialOwnersStep.error.certify',
-        };
-
-        this.getErrors = () => ReimbursementAccountUtils.getErrors(this.props);
-        this.clearError = inputKey => ReimbursementAccountUtils.clearError(this.props, inputKey);
-        this.clearErrors = inputKeys => ReimbursementAccountUtils.clearErrors(this.props, inputKeys);
-        this.getErrorText = inputKey => ReimbursementAccountUtils.getErrorText(this.props, this.errorTranslationKeys, inputKey);
     }
 
     /**
-     * @returns {Boolean}
+     * @param {Object} values - input values passed by the Form component
+     * @returns {Object}
      */
-    validate() {
-        let beneficialOwnersErrors = [];
+    validate(values) {
+        const errors = {};
+
         if (this.state.hasOtherBeneficialOwners) {
-            beneficialOwnersErrors = _.map(this.state.beneficialOwners, ValidationUtils.validateIdentity);
+            _.each(this.state.beneficialOwners, (ownerKey) => {
+                if (!ValidationUtils.isRequiredFulfilled(values[`beneficialOwner.${ownerKey}.firstName`])) {
```

Comment:
Instead of all these isRequiredFulfilled, maybe we can do a for loop? Something like that?

```js
const errorKeys = {
   street: 'address',
   city: 'addressCity',
   state: 'addressState'
};
const requiredFields = ['firstName', 'lastName', 'dob', 'ssnLast4', 'street', 'city', 'zipCode', 'state'];
_.each(requiredFields, (inputKey) => {
   if (!ValidationUtils.isRequiredFulfilled(values[`beneficialOwner.${ownerKey}.${inputKey}`])) {
       const errorKey = errorKeys[inputKey] || inputKey;
       errors[`beneficialOwner.${ownerKey}.${inputKey}`] = this.props.translate(`bankAccount.error.${errorKey}`);
   }
});
```